src/app/freelancer/market/page.tsx

```diff
@@ -78,6 +79,74 @@ interface ProjectsDomain {
   label: string;
 }
 
+const onboardingSteps: Step[] = [
+  {
+    target: '.Dashboard', // Target search bar
+    content: 'Here you can see your active project stats.',
```

Comment:
content: 'Here you can see your active project stats' and comment ' // Target search bar' is added to multiple targets, please fix this